dynamiq/nodes/tools/jina.py

```diff
@@ -0,0 +1,246 @@
+import enum
+from typing import Any, ClassVar, Literal
+
+from pydantic import BaseModel, ConfigDict, Field
+
+from dynamiq.connections import Jina
+from dynamiq.nodes import NodeGroup
+from dynamiq.nodes.node import ConnectionNode, ensure_config
+from dynamiq.runnables import RunnableConfig
+from dynamiq.utils.logger import logger
+
+
+class JinaScrapeInputSchema(BaseModel):
+    url: str = Field(default="", description="Parameter to provide a url of the page to scrape.")
+
+
+class ResponseFormat(str, enum.Enum):
+    DEFAULT = "default"
+    MARKDOWN = "markdown"
+    HTML = "html"
+    TEXT = "text"
+    SCREENSHOT = "screenshot"
+    PAGESHOT = "pageshot"
+
+
+SCRAPE_PATH = "https://r.jina.ai/"
+SEARCH_PATH = "https://s.jina.ai/"
+
+
+class JinaScrapeTool(ConnectionNode):
+    """
+    A tool for scraping web pages, powered by Jina.
+
+    This class is responsible for scraping the content of a web page using Jina.
+
+    Attributes:
+        group (Literal[NodeGroup.TOOLS]): The group to which this tool belongs.
+        name (str): The name of the tool.
+        description (str): A brief description of the tool.
+        connection (Jina): The connection instance for the Jina API.
+        url (str): The url of the page to scrape.
+        timeout(int): The timeout of the scraping process.
+        input_schema (JinaSearchInputSchema): The input schema for the tool.
+    """
+
+    group: Literal[NodeGroup.TOOLS] = NodeGroup.TOOLS
+    name: str = "Jina Scraper Tool"
+    description: str = (
+        "A tool for scraping web pages, powered by Jina. " "You can use this tool to scrape the content of a web page."
+    )
+    response_format: ResponseFormat = ResponseFormat.DEFAULT
+    connection: Jina
+    url: str | None = None
```

Comment:
`url` shoudl be as contant here with a predefined value. So user should change or define it